test/integration/etcd_storage_path_test.go

```diff
@@ -368,75 +513,105 @@ var ephemeralWhiteList = createEphemeralWhiteList(
+	gvr("authorization.openshift.io", "v1", "clusterroles"),
 	gvr("", "v1", "rolebindings"),
+	gvr("authorization.openshift.io", "v1", "rolebindings"),
 	gvr("", "v1", "clusterrolebindings"),
+	gvr("authorization.openshift.io", "v1", "clusterrolebindings"),
 
 	// SAR objects that are not stored in etcd
 	gvr("", "v1", "subjectrulesreviews"),
+	gvr("authorization.openshift.io", "v1", "subjectrulesreviews"),
 	gvr("", "v1", "selfsubjectrulesreviews"),
+	gvr("authorization.openshift.io", "v1", "selfsubjectrulesreviews"),
 	gvr("", "v1", "subjectaccessreviews"),
+	gvr("authorization.openshift.io", "v1", "subjectaccessreviews"),
 	gvr("", "v1", "resourceaccessreviews"),
+	gvr("authorization.openshift.io", "v1", "resourceaccessreviews"),
 	gvr("", "v1", "localsubjectaccessreviews"),
+	gvr("authorization.openshift.io", "v1", "localsubjectaccessreviews"),
 	gvr("", "v1", "localresourceaccessreviews"),
+	gvr("authorization.openshift.io", "v1", "localresourceaccessreviews"),
 	gvr("", "v1", "ispersonalsubjectaccessreviews"),
+	gvr("authorization.openshift.io", "v1", "ispersonalsubjectaccessreviews"),
 	gvr("", "v1", "resourceaccessreviewresponses"),
+	gvr("authorization.openshift.io", "v1", "resourceaccessreviewresponses"),
 	gvr("", "v1", "subjectaccessreviewresponses"),
+	gvr("authorization.openshift.io", "v1", "subjectaccessreviewresponses"),
 	// --
 
 	// github.com/openshift/origin/pkg/build/api/v1
 
 	// used for streaming build logs from pod, not stored in etcd
 	gvr("", "v1", "buildlogs"),
+	gvr("build.openshift.io", "v1", "buildlogs"),
 	gvr("", "v1", "buildlogoptionses"),
+	gvr("build.openshift.io", "v1", "buildlogoptionses"),
 
 	// BuildGenerator helpers not stored in etcd
 	gvr("", "v1", "buildrequests"),
+	gvr("build.openshift.io", "v1", "buildrequests"),
 	gvr("", "v1", "binarybuildrequestoptionses"),
+	gvr("build.openshift.io", "v1", "binarybuildrequestoptionses"),
+	gvr("build.openshift.io", "v1", "statuses"),
 	// --
 
 	// github.com/openshift/origin/pkg/deploy/api/v1
 
 	// used for streaming deployment logs from pod, not stored in etcd
 	gvr("", "v1", "deploymentlogs"),
+	gvr("apps.openshift.io", "v1", "deploymentlogs"),
 	gvr("", "v1", "deploymentlogoptionses"),
+	gvr("apps.openshift.io", "v1", "deploymentlogoptionses"),
 
-	gvr("", "v1", "deploymentrequests"),        // triggers new dc, not stored in etcd
-	gvr("", "v1", "deploymentconfigrollbacks"), // triggers rolleback dc, not stored in etcd
-	// --
+	gvr("", "v1", "deploymentrequests"),                         // triggers new dc, not stored in etcd
+	gvr("apps.openshift.io", "v1", "deploymentrequests"),        // triggers new dc, not stored in etcd
+	gvr("", "v1", "deploymentconfigrollbacks"),                  // triggers rolleback dc, not stored in etcd
+	gvr("apps.openshift.io", "v1", "deploymentconfigrollbacks"), // triggers rolleback dc, not stored in etcd
 
-	// github.com/openshift/origin/pkg/image/api/docker10
-	gvr("", "1.0", "dockerimages"), // part of imageapiv1.Image
```

Comment:
What happened to this and the other `dockerimages` ? I don't see that test anymore